doc/modules/mod_aws_sns.md

```diff
@@ -1,52 +1,3 @@
-* **pm_messages_topic** (string, default: unset) - Defines Amazon SNS Topic for private messages notifications. Remove this option to disable those notifications.
-* **muc_messages_topic** (string, default: unset) - Defines Amazon SNS Topic for group messages notifications. Remove this option to disable those notifications.
-* **plugin_module** (atom, default: 'mod_aws_sns_defaults') - Sets a callback module used for creating user's GUID used in notifications (from user's JID) and for defining custom attributes attached to a published SNS message.
-* **muc_host** (string, default: "conference.@HOST@") - Messages from this MUC host will be sent to the set SNS topic for MUCs.
-* **sns_host** (string, default: unset) - URL to the Amazon SNS service. The URL may be in [virtual host form][aws-virtual-host], and for AWS needs to point at a specific regional endpoint. The scheme, port and path specified in the URL will be used to publish notifications via HTTP POST method.
-* **region** (string, default: unset) - The [AWS region][aws-region] to use for requests.
-* **access_key_id** (string, default: unset) - [ID of the access key][aws-keys] to use for authorization.
-* **secret_access_key** (string, default: unset) - [Secret access key][aws-keys] to use for authorization.
-* **account_id** (string, default: unset) - 12 digit number as defined in [AWS Account Identifiers][aws-acct-identifier] to use for creating TopicArn for publishing notifications.
-* **pool_size** (integer, default: 100) - Worker pool size for publishing notifications
-* **publish_retry_count** (integer, default: 2) - Retry count in case of a publish error
-* **publish_retry_time_ms** (integer, default: 50) - Base exponential backoff time (in ms) for publish errors
-
-[aws-acct-identifier]: http://docs.aws.amazon.com/general/latest/gr/acct-identifiers.html
-[aws-virtual-host]: https://docs.aws.amazon.com/AmazonS3/latest/dev/VirtualHosting.html
-[aws-region]: https://docs.aws.amazon.com/general/latest/gr/rande.html?shortFooter=true#s3_region
-[aws-keys]: https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html?shortFooter=true#access-keys-and-secret-access-keys
-[aws-arn]: http://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html
-
-### Example configuration
-
-```Erlang
-{mod_aws_sns, [
-    {access_key_id, "AKIAIOSFODNN7EXAMPLE"},
-    {secret_access_key, "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"},
-    {region, "eu-west-1"},
-    {account_id, "123456789012"},
-    {sns_host, "sns.eu-west-1.amazonaws.com"},
-    {plugin_module, mod_aws_sns_defaults},
-    {presence_updates_topic, "user_presence_updated"},
-    {pm_messages_topic, "user_message_sent"},
-    {muc_messages_topic, "user_messagegroup_sent"},
-    {pool_size, 100},
-    {publish_retry_count, 2}, 
-    {publish_retry_time_ms, 50}
-   ]}.
-```
+This module is deprecated. Instead, please refer to documentation of [SNS backend of mod_event_pusher](./mod_event_pusher_sns.md).
```

Comment:
This module is deprecated. Please refer to documentation of [SNS backend of mod_event_pusher](./mod_event_pusher_sns).